mlflow/tracking/fluent.py

```diff
@@ -380,6 +380,18 @@ def start_run(
             # if `log_system_metrics` is not specified, we will check environment variable.
             log_system_metrics = MLFLOW_ENABLE_SYSTEM_METRICS_LOGGING.get()
         if log_system_metrics:
+            # Ensure psutil is installed. It was moved to an optional dependency, as it doesn't
+            # have binary for Arm64 Linux and requires build from CPython which was a headache.
```

Comment:
```suggestion
            # have binary for Arm64 Linux and requires build from CPython which is a headache.
```

`was a headache` sounds `it is not now`.